index.php

```diff
@@ -1,7 +1,7 @@
 <?php
 declare( strict_types = 1 );
 
-require_once __DIR__ . '/vendor/autoload.php' );
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
I didn't notice this in #27, but I'm surprised the linter couldn't spot it 🤔 

![image](https://user-images.githubusercontent.com/1218433/86558025-7b701300-bf9b-11ea-9915-8f1b7e2ff545.png)

I searched for a rule to enable for this, but couldn't find one.

Reply:
This is something that can be caught by psalm – I've opened #30 to add this.